src/components/Icons/CatalogScreenActionIcon.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TouchableOpacity, ActionSheetIOS } from 'react-native';
+import { Entypo } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+
+import Colors from '../../constants/Colors';
+
+const CatalogScreenActionIcon = () => {
+  const handlePress = () => {
+    ActionSheetIOS.showActionSheetWithOptions(
+      {
+        options: ['Cancel', 'Placeholder Action', 'Placeholder Action', 'Placeholder Action'],
+        destructiveButtonIndex: 3,
+        cancelButtonIndex: 0,
+        userInterfaceStyle: 'dark',
+      },
+      (buttonIndex) => {
+        if (buttonIndex === 0) {
+          // cancel action
+        } else if (buttonIndex === 1) {
+          // renameAlert();
+        } else if (buttonIndex === 2) {
+          // duplicateAlert();
+        } else if (buttonIndex === 3) {
+          // if (collectionsCount <= 1) {
+          //   deleteErrorAlert();
+          // } else {
+          //   deleteCollectionAlert();
+          // }
+        }
+      },
+    );
+  };
+
+  return (
+    <TouchableOpacity onPress={handlePress}>
+      <Entypo name="dots-three-vertical" size={24} color={Colors.darkgrey} />
```

Comment:
wonder if `CollectionRowActionIcon` should also use `dots-three-vertical` -- unless horizontal is a convention? (for actionsheets?) 

![image](https://user-images.githubusercontent.com/3383704/113577383-2ecf6c00-95ef-11eb-94ab-cad102de1c89.png)

Reply:
This is just a design choice. I thought about giving some visual distinction for the 2 different ActionSheet menus. But since the top-level "Catalog/Collection" view setting is going to be a modal instead, that may justify it not being a triple-dots at all.